helpers/utils

```diff
@@ -329,6 +333,22 @@ ynh_setup_source() {
             unzip -quo $src_filename -d "$dest_dir"
         fi
         ynh_secure_remove --file="$src_filename"
+    elif [[ "$src_format" =~ ^gz|xz|bz2$ ]]; then
+        if [[ -z "$src_rename" ]]; then
+            if [[ "$source_id" == "main" ]]; then
+                local src_rename=$app
```

Comment:
I'm confused by this because this will lead to inconsistent behavior between this case and other case (where the main source is just kept as "main" and not "$app") x_X ... What's the rationale for this ?

Reply:
The goal is to allow the omission of `rename` in resource definition. (I just remember, writing this line, I've mentioned to set `rename` in the «how to test» section of this PR 🥴)

In the the forgejo app case, I thought it was logical to extract the archive to the same name as the application one.